src/app/mailing-list/mailing-list.component.ts

```diff
@@ -155,22 +155,15 @@ export class MailingListComponent implements OnInit {
 
   getPossibleInfoColumns( contact: MailingListContact ) {
     if (contact != null && contact.info != null) {
-      let o: any = JSON.parse( contact.info );
-      if (o != null) {
-        let k: string[] = Object.keys( o );
+        let k: string[] = Object.keys( contact );
```

Comment:
ok to lose the ".info" part here?

Reply:
@SimoneMaiwald `contact` which is supplied to the method `getPossibleInfoColumns` is already parsed and has a type `MailingListContact` , so json parsing on 159-th line was just obsolete in this case, that's why I removed it.